web/xfa_layer_builder.css

```diff
@@ -59,7 +59,7 @@
 }
 
 .xfaLayer ::selection {
-  background: rgba(0, 0, 255, 1);
+  background: AccentColor;
```

Comment:
The pre-processor suggestion, see above, applies here as well.

---

Also, is there not an `opacity` value that needs to be changed in this file as well?